src/hoc/Layout/Layout.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+
+import Aux from "../Aux/Aux";
+
+import fire from "../../API/config/Fire";
+
+import Welcome from "../../containers/Welcome";
+// import Login from "../../containers/Authentication/Login";
+
+import Home from "../../containers/Tasks/Home/index";
+
+class Layout extends Component {
```

Comment:
The whole component logic and file and be removed and replaced in app.js

As a fellow developer when I go to your app.js it's helpful if I can see the auth logic to conditionally render  the app. 
This is a little over engineered

Reply:
agreed, replaced to app.js